Listing '.'...
Listing './bin'...
Listing './include'...
Listing './lib'...
Listing './lib/python3.5'...
Listing './lib/python3.5/site-packages'...
Listing './lib/python3.5/site-packages/astroid'...
Listing './lib/python3.5/site-packages/astroid/brain'...
Listing './lib/python3.5/site-packages/astroid/tests'...
Listing './lib/python3.5/site-packages/astroid/tests/testdata'...
Listing './lib/python3.5/site-packages/astroid/tests/testdata/python2'...
Listing './lib/python3.5/site-packages/astroid/tests/testdata/python2/data'...
Listing './lib/python3.5/site-packages/astroid/tests/testdata/python2/data/SSL1'...
Compiling './lib/python3.5/site-packages/astroid/tests/testdata/python2/data/SSL1/Connection1.py'...
***   File "./lib/python3.5/site-packages/astroid/tests/testdata/python2/data/SSL1/Connection1.py", line 14
    print 'init Connection'
                          ^
SyntaxError: Missing parentheses in call to 'print'

Listing './lib/python3.5/site-packages/astroid/tests/testdata/python2/data/absimp'...
Listing './lib/python3.5/site-packages/astroid/tests/testdata/python2/data/absimp/sidepackage'...
Compiling './lib/python3.5/site-packages/astroid/tests/testdata/python2/data/all.py'...
***   File "./lib/python3.5/site-packages/astroid/tests/testdata/python2/data/all.py", line 7
    def func(): print 'yo'
                         ^
SyntaxError: invalid syntax

Listing './lib/python3.5/site-packages/astroid/tests/testdata/python2/data/appl'...
Compiling './lib/python3.5/site-packages/astroid/tests/testdata/python2/data/appl/myConnection.py'...
***   File "./lib/python3.5/site-packages/astroid/tests/testdata/python2/data/appl/myConnection.py", line 7
    print 'MyConnection init'
                            ^
SyntaxError: Missing parentheses in call to 'print'

Listing './lib/python3.5/site-packages/astroid/tests/testdata/python2/data/find_test'...
Listing './lib/python3.5/site-packages/astroid/tests/testdata/python2/data/lmfp'...
Compiling './lib/python3.5/site-packages/astroid/tests/testdata/python2/data/module.py'...
***   File "./lib/python3.5/site-packages/astroid/tests/testdata/python2/data/module.py", line 22
    print '!!!'
              ^
SyntaxError: Missing parentheses in call to 'print'

Listing './lib/python3.5/site-packages/astroid/tests/testdata/python2/data/module1abs'...
Compiling './lib/python3.5/site-packages/astroid/tests/testdata/python2/data/module1abs/__init__.py'...
***   File "./lib/python3.5/site-packages/astroid/tests/testdata/python2/data/module1abs/__init__.py", line 4
    print sys.version
            ^
SyntaxError: Missing parentheses in call to 'print'

Compiling './lib/python3.5/site-packages/astroid/tests/testdata/python2/data/module2.py'...
***   File "./lib/python3.5/site-packages/astroid/tests/testdata/python2/data/module2.py", line 73
    assert `1`
           ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/astroid/tests/testdata/python2/data/noendingnewline.py'...
***   File "./lib/python3.5/site-packages/astroid/tests/testdata/python2/data/noendingnewline.py", line 21
    print 'a'
            ^
SyntaxError: Missing parentheses in call to 'print'

Compiling './lib/python3.5/site-packages/astroid/tests/testdata/python2/data/nonregr.py'...
***   File "./lib/python3.5/site-packages/astroid/tests/testdata/python2/data/nonregr.py", line 16
    print v.get('yo')
          ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/astroid/tests/testdata/python2/data/notall.py'...
***   File "./lib/python3.5/site-packages/astroid/tests/testdata/python2/data/notall.py", line 7
    def func(): print 'yo'
                         ^
SyntaxError: invalid syntax

Listing './lib/python3.5/site-packages/astroid/tests/testdata/python2/data/package'...
Listing './lib/python3.5/site-packages/astroid/tests/testdata/python2/data/package/subpackage'...
Listing './lib/python3.5/site-packages/astroid/tests/testdata/python2/data/unicode_package'...
Listing './lib/python3.5/site-packages/astroid/tests/testdata/python2/data/unicode_package/core'...
Listing './lib/python3.5/site-packages/astroid/tests/testdata/python3'...
Listing './lib/python3.5/site-packages/astroid/tests/testdata/python3/data'...
Listing './lib/python3.5/site-packages/astroid/tests/testdata/python3/data/SSL1'...
Listing './lib/python3.5/site-packages/astroid/tests/testdata/python3/data/absimp'...
Listing './lib/python3.5/site-packages/astroid/tests/testdata/python3/data/absimp/sidepackage'...
Listing './lib/python3.5/site-packages/astroid/tests/testdata/python3/data/appl'...
Listing './lib/python3.5/site-packages/astroid/tests/testdata/python3/data/find_test'...
Listing './lib/python3.5/site-packages/astroid/tests/testdata/python3/data/lmfp'...
Listing './lib/python3.5/site-packages/astroid/tests/testdata/python3/data/module1abs'...
Listing './lib/python3.5/site-packages/astroid/tests/testdata/python3/data/package'...
Listing './lib/python3.5/site-packages/astroid/tests/testdata/python3/data/package/subpackage'...
Listing './lib/python3.5/site-packages/astroid/tests/testdata/python3/data/unicode_package'...
Listing './lib/python3.5/site-packages/astroid/tests/testdata/python3/data/unicode_package/core'...
Listing './lib/python3.5/site-packages/astroid-1.3.2.dist-info'...
Listing './lib/python3.5/site-packages/logilab'...
Listing './lib/python3.5/site-packages/logilab/common'...
Listing './lib/python3.5/site-packages/logilab/common/ureports'...
Listing './lib/python3.5/site-packages/logilab_common-1.4.1.egg-info'...
Listing './lib/python3.5/site-packages/pip'...
Listing './lib/python3.5/site-packages/pip/_vendor'...
Listing './lib/python3.5/site-packages/pip/commands'...
Listing './lib/python3.5/site-packages/pip/compat'...
Listing './lib/python3.5/site-packages/pip/models'...
Listing './lib/python3.5/site-packages/pip/operations'...
Listing './lib/python3.5/site-packages/pip/req'...
Listing './lib/python3.5/site-packages/pip/utils'...
Listing './lib/python3.5/site-packages/pip/vcs'...
Listing './lib/python3.5/site-packages/pip-9.0.1.dist-info'...
Listing './lib/python3.5/site-packages/pkg_resources'...
Listing './lib/python3.5/site-packages/pkg_resources/_vendor'...
Listing './lib/python3.5/site-packages/pkg_resources/_vendor/packaging'...
Listing './lib/python3.5/site-packages/pkg_resources/extern'...
Listing './lib/python3.5/site-packages/pkg_resources-0.0.0.dist-info'...
Listing './lib/python3.5/site-packages/pylint'...
Listing './lib/python3.5/site-packages/pylint/checkers'...
Listing './lib/python3.5/site-packages/pylint/pyreverse'...
Listing './lib/python3.5/site-packages/pylint/reporters'...
Listing './lib/python3.5/site-packages/pylint/test'...
Listing './lib/python3.5/site-packages/pylint/test/data'...
Listing './lib/python3.5/site-packages/pylint/test/functional'...
Compiling './lib/python3.5/site-packages/pylint/test/functional/yield_outside_func.py'...
***   File "./lib/python3.5/site-packages/pylint/test/functional/yield_outside_func.py", line 2
    yield 1  # [yield-outside-function]
                                      ^
SyntaxError: 'yield' outside function

Listing './lib/python3.5/site-packages/pylint/test/input'...
Compiling './lib/python3.5/site-packages/pylint/test/input/func_continue_not_in_loop.py'...
***   File "./lib/python3.5/site-packages/pylint/test/input/func_continue_not_in_loop.py", line 8
    continue
    ^
SyntaxError: 'continue' not properly in loop

Compiling './lib/python3.5/site-packages/pylint/test/input/func_e0001_py30.py'...
***   File "./lib/python3.5/site-packages/pylint/test/input/func_e0001_py30.py", line 12
    raise 'exception', 'message'
                     ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/pylint/test/input/func_e0108.py'...
***   File "./lib/python3.5/site-packages/pylint/test/input/func_e0108.py", line 4
    def foo1(_, _):
                  ^
SyntaxError: duplicate argument '_' in function definition

Compiling './lib/python3.5/site-packages/pylint/test/input/func_keyword_repeat.py'...
***   File "./lib/python3.5/site-packages/pylint/test/input/func_keyword_repeat.py", line 8
    function_default_arg(two=5, two=7)
                               ^
SyntaxError: keyword argument repeated

Compiling './lib/python3.5/site-packages/pylint/test/input/func_return_outside_func.py'...
***   File "./lib/python3.5/site-packages/pylint/test/input/func_return_outside_func.py", line 3
    return
         ^
SyntaxError: 'return' outside function

Compiling './lib/python3.5/site-packages/pylint/test/input/func_syntax_error.py'...
***   File "./lib/python3.5/site-packages/pylint/test/input/func_syntax_error.py", line 1
    def toto
           ^
SyntaxError: invalid syntax

Listing './lib/python3.5/site-packages/pylint/test/input/func_w0401_package'...
Compiling './lib/python3.5/site-packages/pylint/test/input/func_w0705.py'...
***   File "./lib/python3.5/site-packages/pylint/test/input/func_w0705.py", line 28
    __revision__ += 1
                   ^
SyntaxError: default 'except:' must be last

Compiling './lib/python3.5/site-packages/pylint/test/input/syntax_error.py'...
*** Sorry: IndentationError: expected an indented block (syntax_error.py, line 2)
Listing './lib/python3.5/site-packages/pylint/test/messages'...
Listing './lib/python3.5/site-packages/pylint/test/regrtest_data'...
Listing './lib/python3.5/site-packages/pylint/test/regrtest_data/absimp'...
Listing './lib/python3.5/site-packages/pylint/test/regrtest_data/package'...
Listing './lib/python3.5/site-packages/pylint/test/regrtest_data/package/subpackage'...
Listing './lib/python3.5/site-packages/pylint/test/regrtest_data/package_all'...
Listing './lib/python3.5/site-packages/pylint-1.4.0.dist-info'...
Listing './lib/python3.5/site-packages/pyplantuml'...
Listing './lib/python3.5/site-packages/pyplantuml-0.1.51.egg-info'...
Listing './lib/python3.5/site-packages/setuptools'...
Listing './lib/python3.5/site-packages/setuptools/command'...
Listing './lib/python3.5/site-packages/setuptools/extern'...
Listing './lib/python3.5/site-packages/setuptools-32.3.1.dist-info'...
Listing './lib/python3.5/site-packages/six-1.11.0.dist-info'...
Listing './share'...
Listing './share/python-wheels'...
